public/mocks/v1_get_actor_node_attributes.json

```diff
@@ -595,84 +595,142 @@
       }
     ],
     "companies_exporting": {
-      "dimensions_y": {
-        "name": "Soy exported in 2015",
-        "unit": "TN"
-      },
       "dimensions_x": [
         {
-          "name": "Land use",
-          "unit": null
-        },
-        {
           "name": "Territorial Deforestation",
-          "unit": null
+          "chart_name": "Territorial<br/>Deforestation",
```

Comment:
I would remove `chart_name`. It is redundant with `name`. Line breaks are purely presentationbal and should be dealt with CSS.